src/Broadway/EventStore/DBALEventStore.php

```diff
@@ -45,12 +49,14 @@ public function __construct(
         Connection $connection,
         SerializerInterface $payloadSerializer,
         SerializerInterface $metadataSerializer,
-        $tableName
+        $tableName,
+        $useBinary = false
     ) {
         $this->connection         = $connection;
         $this->payloadSerializer  = $payloadSerializer;
         $this->metadataSerializer = $metadataSerializer;
         $this->tableName          = $tableName;
+        $this->useBinary          = Type::hasType('binary') && (bool) $useBinary;
```

Comment:
Can we validate/assert this instead of silently calling back to the default? Maybe also validate it in the Extension?